src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Account.ExternalLogin.cs.cshtml

```diff
@@ -142,6 +147,10 @@ namespace @thisNamespace
             if (ModelState.IsValid)
             {
                 var user = new @(Model.UserClass) { UserName = Input.Email, Email = Input.Email };
+
```

Comment:
You don't need these changes the scaffolded code can use the POCO itself, this hacky stuff is due to weirdness in the UI package, we want to generate new User() code, so we don't want these changes

Reply:
We only want the ProviderDisplayName change in this file

Reply:
reverted other than ProviderDisplayName